lib/tasks/test_track_rails_client_tasks.rake

```diff
@@ -19,7 +19,7 @@ namespace :test_track do
   end
 
   desc 'Sets an environment variable to block build timestamp generation on application initialization'
-  task :skip_load_build_timestamp do
+  task :skip_load_build_timestamp do # rubocop:disable Rails/RakeEnvironment
     ENV['SKIP_TESTTRACK_SET_BUILD_TIMESTAMP'] = '1'
```

Comment:
I wasn't sure if we intentionally didn't inherit from `:environment` here -- cc @phantomphildius

Reply:
This was when we were trying to avoid loading all the other tasks that would have gotten called via environment. We can probably add it back now!

Reply:
Cool, will add it back.

Reply:
does this seem right @aburgel ?